openmetadata-ui/src/main/resources/ui/src/components/DataAssets/DataAssetsHeader/DataAssetsHeader.component.tsx

```diff
@@ -116,19 +124,27 @@ export const ExtraInfoLink = ({
   href,
   newTab = false,
   ellipsis = false,
+  isNewDesign = false,
```

Comment:
We don't need this.
This is used at one place only and we want to redesign, so remove this param